examples/transfer/plot_bci_example.py

```diff
@@ -0,0 +1,199 @@
+from sklearn.model_selection import StratifiedShuffleSplit
+import matplotlib.pyplot as plt
+
+from mne import Epochs, pick_types, events_from_annotations
+from mne.io import concatenate_raws
+from mne.io.edf import read_raw_edf
+from mne.datasets import eegbci
+from mne import set_log_level
+
+from pyriemann.classification import MDM
+from pyriemann.estimation import Covariances
+from pyriemann.transfer import (
+    encode_domains,
+    TLDummy,
+    TLCenter,
+    TLClassifier,
+    TLSplitter,
+)
+
+set_log_level(verbose=False)
+
+
+###############################################################################
+
+def get_subject_dataset(subject):
+
+    # Consider epochs that start 1s after cue onset.
+    tmin, tmax = 1., 2.
+    event_id = dict(hands=2, feet=3)
+    runs = [6, 10, 14]  # motor imagery: hands vs feet
+
+    # Download data with MNE
+    raw_files = [
+        read_raw_edf(f, preload=True) for f in eegbci.load_data(subject, runs)
+    ]
+    raw = concatenate_raws(raw_files)
+
+    # Select only EEG channels
+    picks = pick_types(
+        raw.info, meg=False, eeg=True, stim=False, eog=False, exclude='bads')
+    # select only nine electrodes: F3, Fz, F4, C3, Cz, C4, P3, Pz, P4
+    picks = picks[[31, 33, 35, 8, 10, 12, 48, 50, 52]]
+
+    # Apply band-pass filter
+    raw.filter(7., 35., method='iir', picks=picks)
+
+    # Check the events
+    events, _ = events_from_annotations(raw, event_id=dict(T1=2, T2=3))
+
+    # Define the epochs
+    epochs = Epochs(
+        raw,
+        events,
+        event_id,
+        tmin,
+        tmax,
+        proj=True,
+        picks=picks,
+        baseline=None,
+        preload=True,
+        verbose=False)
+
+    # Extract the labels for each event
+    labels = epochs.events[:, -1] - 2
+
+    # Compute covariance matrices on scaled data
+    covs = Covariances().fit_transform(1e6 * epochs.get_data())
+
+    return covs, labels
+
+
+###############################################################################
+
+# We will consider subjects from the Physionet EEG database for which the
+# intra-subject classification has been checked to be > 0.70
+subject_list = [1, 2, 4, 7, 8, 15, 20, 29, 34, 35]
+
+# Load the data from subjects
+X, y, d = [], [], []
+for i, subject_source in enumerate(subject_list):
+    X_source_i, y_source_i = get_subject_dataset(subject=subject_source)
+    X.append(X_source_i)
+    y.append(y_source_i)
+    d = d + [f'subject_{subject_source:02}'] * len(X_source_i)
+X = np.concatenate(X)
+y = np.concatenate(y)
+domains = np.array(d)
+
+# Encode the data for transfer learning purposes
+X_enc, y_enc = encode_domains(X, y, domains)
+
+# Object for splitting the datasets into training and validation partitions
+n_splits = 5  # How many times to split the target domain into train/test
+cv = TLSplitter(
+    target_domain='',
+    cv_iterator=StratifiedShuffleSplit(
+        n_splits=n_splits, train_size=0.10, random_state=42))
```

Comment:
```suggestion
cv = TLSplitter(
    target_domain='',
    cv=StratifiedShuffleSplit(
        n_splits=n_splits, train_size=0.10, random_state=42))
```

I don't really like inventing a new param name for what is just a standard sklearn cv object. I would keep the param name `cv` and just do

`tl_cv = TLSplitter(`

if you want to be clear that this is a cv object for transfer